lib/version/make_version.cmake.in

```diff
@@ -45,7 +45,7 @@ execute_process(
 )
 
 execute_process(
-  COMMAND python ${SOURCE_DIR}/python/print_kotekan_version.py
+  COMMAND python ${SOURCE_DIR}/python/kotekan/print_kotekan_version.py
   WORKING_DIRECTORY ${SOURCE_DIR}
   OUTPUT_VARIABLE KOTEKAN_VERSION
   OUTPUT_STRIP_TRAILING_WHITESPACE
```

Comment:
I think this would work and allow you to eliminate the `print_kotekan_version.py` script entirely.

```suggestion
  COMMAND python -c "import kotekan; print(kotekan.__version__)"
  WORKING_DIRECTORY ${SOURCE_DIR}/python/
  OUTPUT_VARIABLE KOTEKAN_VERSION
  OUTPUT_STRIP_TRAILING_WHITESPACE
```

Reply:
I am only 98% sure this would work. So it's probably worth testing.

Reply:
I've tested your changes and they work